### PR TITLE
doc: stylish dispatchers.md, #24717

### PR DESCRIPTION
**Pull request by patriknw**
Refs #24717

### CODE REVIEW

**Comment by patriknw** [comment]
PLS BUILD

**Review by helena — APPROVED** [review]
LGTM

File: akka-docs/src/test/java/jdocs/actor/typed/PrintActor.java [hunk 1]
```diff
@@ -10,6 +10,10 @@
 // #print-actor
 class PrintActor extends AbstractBehavior<Integer> {
 
+  public static Behavior<Integer> create() {
+    return Behaviors.setup(notUsed -> new PrintActor());
```
Comment (johanandren) [inline akka-docs/src/test/java/jdocs/actor/typed/PrintActor.java]: In this kind of place I'd rather call it `context` even if it isn't used.

**Review by johanandren — APPROVED** [review]
LGTM with one opinion

File: akka-docs/src/test/java/jdocs/actor/typed/PrintActor.java (hunk 1 above)
Comment (patriknw) [reply akka-docs/src/test/java/jdocs/actor/typed/PrintActor.java]: okidok, I agree

**Pull request merged**